taxi/templates/includes/sidebar.html

```diff
@@ -0,0 +1,6 @@
+<ul>
+	<li> <a href="{%  url 'taxi:index' %}">Home page</a></li>
+	<li><a href="#">Manufacturers</a></li>
+	<li><a href="#">Cars</a></li>
+	<li><a href="#">Drivers</a></li>
```

Comment:
The links for 'Manufacturers', 'Cars', and 'Drivers' currently use '#' as the href value, which is a placeholder. Replace these with the appropriate URL template tags, such as `{% url 'app_name:view_name' %}`, to ensure they navigate to the correct pages.